### PR TITLE
feat(ApplicationEventBridgeRule): expose scheduleExpression

### PR DESCRIPTION
**Pull request by kschelonka**
# Goal

Expose `scheduleExpression` for ApplicationEventBridgeRule

Tickets:
* Pre-requesite for [INFRA-690]

[INFRA-690]: https://getpocket.atlassian.net/browse/INFRA-690?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/base/ApplicationEventBridgeRule.ts [hunk 1]
```diff
@@ -39,13 +48,15 @@ export class ApplicationEventBridgeRule extends Resource {
       throw new Error('AWS allows only up to 5 targets per event bridge rule');
     }
     const eventBus = this.config.eventBusName ?? 'default';
+    const { scheduleExpression, eventPattern } = this.config;
```
Comment (jpetto) [inline src/base/ApplicationEventBridgeRule.ts:51]: do we need to explicitly check in code that one of the above is defined? if neither are defined, will this fail to deploy?

on that note, should we add a test when neither are provided?

(not sure how restrictive/guardrail-y we want to be here.)

**Review by jpetto — APPROVED** [review]
no blockers, but a question on validating/testing the new config.

File: src/base/ApplicationEventBridgeRule.ts (hunk 1 above)
Comment (kschelonka) [reply src/base/ApplicationEventBridgeRule.ts:51]: checking if it will synth

Comment (kschelonka) [reply src/base/ApplicationEventBridgeRule.ts:51]: The error is thrown when you attempt to plan or validate the terraform file. 

```

Error: Missing required argument

  on cdk.tf.json line 37, in resource.aws_cloudwatch_event_rule.acmeexample_testeventbridgerule_9743ED72:
  37:       }

"event_pattern": one of `event_pattern,schedule_expression` must be specified
```

**Pull request merged**